katas/001_optimize_a_query.md

```diff
@@ -20,9 +20,9 @@ Since you have already gone through the Movie Demo setup, we'll need to set up a
 
 Go ahead and start a new case by clicking _Relevancy Cases_ drop down and choosing _Create a Case_.  
 
-Let's call the case _Notebook Computers_.   Then, instead of the default Solr instance, let's go ahead and use our Chorus Electronics index using this URL (notice the embedded user credentials):
```

Comment:
great clean ups from when I rolled in Keycloak and security for everything except the /select end point!